contract-tools/xchain/xchain.go

```diff
@@ -981,12 +993,13 @@ func loadPrivateKey(cfg *Config) (*bind.TransactOpts, error) {
 
 	// Import existing key
 	a, err := ks.Import(keyJSON, os.Getenv("XCHAIN_PASSPHRASE"), os.Getenv("XCHAIN_PASSPHRASE"))
-	if err != nil {
-		return nil, fmt.Errorf("failed to import key %s: %w", cfg.ClientAddr, err)
-	}
 	if err := ks.Unlock(a, os.Getenv("XCHAIN_PASSPHRASE")); err != nil {
 		return nil, fmt.Errorf("failed to unlock keystore: %w", err)
 	}
+	if err != nil {
```

Comment:
Hmm why move this error checking below ?

Reply:
Ah! sorry, was doing some debugging and interpreted that `:=` as jsut `=` leading to this, correcting it.